src/lib/index.ts

```diff
@@ -59,10 +59,40 @@ export function makeSubtitle(html: string): string | null {
 		: firstParagraph;
 }
 
-const authorTimestampFormat = "HH:mm EEE d MMMM yyyy"
+function ordinal_suffix_of(i) {
+    let j = i % 10,
+        k = i % 100;
+    if (j === 1 && k !== 11) {
+        return i + "st";
+    }
+    if (j === 2 && k !== 12) {
+        return i + "nd";
+    }
+    if (j === 3 && k !== 13) {
+        return i + "rd";
+    }
+    return i + "th";
+}
+
+function numberToOrdinal(i) {
```

Comment:
`i` implicitly has an `any` type here, needs to be annotated. Also is it possible to make this more concise? Consider:
https://github.com/moment/luxon/issues/118#issuecomment-781192208

Reply:
We could make it shorter, but does it really need to be much more concise? It's fairly readable atm which I don't think is the case for the one you linked